archinstall/lib/user_interaction.py

```diff
@@ -22,11 +22,7 @@ def get_longest_option(options):
 def check_for_correct_username(username):
 	if re.match(r'^[a-z_][a-z0-9_-]*\$?$', username) and len(username) <= 32:
 		return True
-	log(
-		"The username you entered is invalid. Try again",
-		level=LOG_LEVELS.Warning,
```

Comment:
Removing the `level=LOG_LEVELS.Warning` here will cause `systemd-journal` output to not be properly color coded. Perhaps this is a non-issue? Not sure how many people check the journal logs for errors :P

Reply:
I mean I check them for errors all the time when troubleshooting servers ;) As far as color coding goes, it does make it easier to spot it.

Reply:
As I can see through the code, other logs with warnings like this didn't have a `level` parameter

Reply:
Ah, yea that's probably my bad on omitting the level in the other places hehe.